eng/pipelines/common/xplat-setup.yml

```diff
@@ -103,12 +103,12 @@ jobs:
     ${{ if eq(parameters.jobParameters.pool, '') }}:
       pool:
         # Public Linux Build Pool
-        ${{ if and(in(parameters.osGroup, 'Linux', 'FreeBSD'), eq(variables['System.TeamProject'], 'public')) }}:
+        ${{ if and(in(parameters.osGroup, 'Linux', 'FreeBSD', 'Android'), eq(variables['System.TeamProject'], 'public')) }}:
```

Comment:
Let's revert this change. Android builds inside a docker container so let's use the hosted pools which is effectively by leaving this as it was.

Reply:
Yeah, you're right forgot about the size constraint for Android. I think we should definitely improve that though